### ISSUE TITLE
RMSEWithUncertainty loss on GPU?

### ISSUE THREAD

**Issue by jarandaf** [catboost/catboost#1573]
Is it supported? I get the following error:

```bash
CatBoostError: catboost/cuda/train_lib/train.cpp:265: Error: optimization scheme is not supported for GPU learning Loss=RMSEWithUncertainty;OptimizationScheme=SymmetricTree
```

I have tried different tree growing policies but the error still remains.

**Comment by LyzhinIvan** [comment]
Hi, @jarandaf! RMSEWithUncertainty is not supported on GPU and not planned for next releases yet.

**Comment by jarandaf** [comment]
Thank you @LyzhinIvan for your answer.

**Comment by niklashoelter** [comment]
Since this issue is already more than 3 years old - are there any updates/future plans on adding GPU support for RMSEWithUncertainty?

**Comment by krds00** [comment]
Any updates? Really useful feature to estimate confidence intervals in cb

**Comment by andrey-khropov** [comment]
RMSEWithUncertainty has been implemented in https://github.com/catboost/catboost/commit/0f0164614d67b977d50ff41833e3f1ecb251a235 and https://github.com/catboost/catboost/commit/158ec34378411cc98230b98486c685c4a0713aad and included in Release 1.2.

I've updated the documentation to indicate that it is supported now.
